python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -145,22 +220,37 @@ def uniform_neighbor_sample(
         indices_t = numpy.int32
 
     if input_graph.renumbered:
-        start_list = input_graph.lookup_internal_vertex_id(start_list).compute()
+        start_list = input_graph.lookup_internal_vertex_id(start_list)
+
+    if isinstance(start_list, cudf.Series):
+        start_list = dask_cudf.from_cudf(start_list, npartitions=len(Comms.get_workers()))
```

Comment:
So this can be quite slow, I suggest only doing this for start_list if it is all ready in dask. 

See PR: https://github.com/rapidsai/cugraph/pull/2887

Reply:
resolved